konig-dao-core/pom.xml

```diff
@@ -34,12 +34,21 @@
 		<artifactId>sesame-model</artifactId>
 		<version>${sesame.version}</version>
 	</dependency>
-	
+	<dependency>
+    <groupId>com.google.cloud</groupId>

```

Comment:
As a general rule, maven projects with names that end with *-core are supposed to be platform independent.
So you shouldn't add a dependency on the Google Cloud Platform to any *-core project.  It's ok to leave it this way for now, but please add an issue to issue tracker so that we can fix it later.